lib/structures/Message.js

```diff
@@ -34,8 +34,6 @@ const User = require("./User");
 * @prop {Boolean} pinned Whether the message is pinned or not
 * @prop {String?} prefix The prefix used in the Message, if any (CommandClient only)
 * @prop {Object} reactions An object containing the reactions on the message
```

Comment:
Probably best if `me` and `count` info was explained here for better clarification instead of just completely removing it